libs/sdk-react-native/android/src/main/java/com/breezsdk/MapResponse.kt

```diff
@@ -0,0 +1,60 @@
+package com.breezsdk
+
+import com.facebook.react.bridge.Arguments
+import com.facebook.react.bridge.WritableArray
+import com.facebook.react.bridge.WritableMap
+
+fun writableMapOf(vararg values: Pair<String, *>): WritableMap {
```

Comment:
Isn't it better to use the readonly version for the return types (ReadableMap)?

Reply:
You're right, these return types should be ReadableArray or ReadableMap.